sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient/FormRecognizerModelFactory.cs

```diff
@@ -505,338 +503,5 @@ public static TrainingDocumentInfo TrainingDocumentInfo(string name, int pageCou
         {
             return new TrainingDocumentInfo(name, pageCount, errors?.ToList(), status, modelId);
         }
-
-        #region generated methods
```

Comment:
These methods we're removing existed only to hide autogenerated members that were placed in the `FormRecognizerModelFactory`. They correspond to models from the new API, but were being added to the old factory.